ui/component/chat/view.jsx

```diff
@@ -26,52 +25,60 @@ export const VIEW_MODES = {
 const COMMENT_SCROLL_TIMEOUT = 25;
 
 type Props = {
+  customViewMode?: string,
   embed?: boolean,
-  isPopoutWindow?: boolean,
-  uri: string,
   hideHeader?: boolean,
   hyperchatsHidden?: boolean,
-  customViewMode?: string,
+  isPopoutWindow?: boolean,
   setCustomViewMode?: (any) => void,
+  uri: string,
   // redux
+  channelId: string,
   claimId?: string,
   comments: Array<Comment>,
-  pinnedComments: Array<Comment>,
-  superChats: Array<Comment>,
   doCommentList: (
     uri: string,
     parentId: ?string,
     page: number,
     pageSize: number,
     sortBy: ?number,
-    isLivestream: boolean
+    isLivestream: boolean,
+    chatCommentsRestrictedToChannelMembers: ?boolean,
+    activeChannelId: ?string
   ) => void,
-  doResolveUris: (uris: Array<string>, cache: boolean) => void,
+  doFetchChannelMembershipsForChannelIds: (channelId: string, claimIds: ClaimIds) => void,
+  doFetchOdyseeMembershipForChannelIds: (claimIds: ClaimIds) => void,
   doHyperChatList: (uri: string) => void,
-  claimsByUri: { [string]: any },
-  doFetchUserMemberships: (claimIdCsv: string) => void,
+  doResolveUris: (uris: Array<string>, cache: boolean) => void,
+  pinnedComments: Array<Comment>,
   setLayountRendered: (boolean) => void,
+  superChats: Array<Comment>,
+  chatCommentsRestrictedToChannelMembers: boolean,
+  activeChannelId: string,
 };
 
 export default function ChatLayout(props: Props) {
   const {
+    activeChannelId,
+    channelId,
+    chatCommentsRestrictedToChannelMembers,
     claimId,
     comments: commentsByChronologicalOrder,
+    customViewMode,
+    doCommentList,
+    doFetchChannelMembershipsForChannelIds,
+    doFetchOdyseeMembershipForChannelIds,
+    doHyperChatList,
+    doResolveUris,
     embed,
-    isPopoutWindow,
-    pinnedComments,
-    superChats: hyperChatsByAmount,
-    uri,
     hideHeader,
     hyperchatsHidden,
-    customViewMode,
+    isPopoutWindow,
+    pinnedComments,
     setCustomViewMode,
-    doCommentList,
-    doResolveUris,
-    doHyperChatList,
-    doFetchUserMemberships,
-    claimsByUri,
     setLayountRendered,
+    superChats: hyperChatsByAmount,
+    uri,
```

Comment:
:small_red_triangle: @mayeaux, I just mentioned yesterday about not combining refactor and development in the same PR?  Can you please follow standard practice like the rest of us?

This is getting impossible to review.